config/locales/en/helpers.yml

```diff
@@ -223,8 +213,8 @@ en:
         conviction_bail: Time spent on bail with an electronic tag affects the spent date of your conviction. Time spent on bail without a tag does not.
       steps_conviction_conviction_bail_days_form:
         conviction_bail_days: This is decided by the court. If you do not know the exact number of days you can enter an approximate number or leave this blank, but it might affect the accuracy of your result.
-      radio_buttons:
-        conviction_subtype:
+      steps_conviction_conviction_subtype_form:
+        conviction_subtype_options:
```

Comment:
Before it was not possible but I think now with the new gem is possible to just remove the hint key when we don't want to show a hint, instead of having an empty string `fine: ""` can you try? That way we clean up a lot of those empty keys in this group.